### ISSUE TITLE
Windows support

### ISSUE THREAD

**Issue by novemberborn** [bcoe/nyc#81]
Ticket to track progress. @bcoe should probably flesh this out more :smiley_cat:

**Comment by sindresorhus** [comment]
Would be nice if you set up Appveyor so you have continuous testing on Windows too.

**Comment by bcoe** [comment]
I've made some further changes to our patched version of `spawn-wrap` and my testing copy of Windows now works with subprocesses \o/ 

Once the [test rewrite](https://github.com/bcoe/nyc/pull/89) is finished, we should try the new test-suite out on Windows and see what additional work would be needed to get things working on AppVeyor.

We should also work on getting on the master branch of `spawn-wrap` again, we've deviated quite a bit:

https://github.com/isaacs/spawn-wrap/pull/5 -- I can try to prod @isaacs, I know he's not super excited to dive back into this codebase :D

CC: @novemberborn, @jamestalmage

**Comment by novemberborn** [comment]
> Once the test rewrite is finished, we should try the new test-suite out on Windows and see what additional work would be needed to get things working on AppVeyor.

Sounds good.

**Comment by jamestalmage** [comment]
@bcoe 

If you want to open up an AppVeyor account I can help track down any issues it introduces (that was one of my first contributions to AVA). 

AppVeyor makes it a bit of a pain to add team members (when compared to Travis), here were the instructions I hunted down and sent to @sindresorhus. I think these worked out OK.

> 1. Click your name in the upper-right for a drop-down
> 2. Choose “Team”
> 3. Choose “Add Role” - name it avaor something.
> 4. Choose some permissions
> 5. Back on the “Team” page add a Collaborator and assign the role
> 6. Go to the project settings and enable the Role

Also, you will need the e-mail I signed up with (which I foolishly I did not do with my public email).

Sent via `gitter`.
